static/js/routes/common.min.js

```diff
@@ -7063,20 +7063,20 @@ return je(this,function(t,n,r){var i;return oe.isWindow(t)?t.document.documentEl
 
 }).call(this,typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {})
-},{"/Users/bradforda/Projects/cfgov-refresh/node_modules/jquery/dist/jquery.js":34}],72:[function(require,module,exports){
+},{"/Users/wilsong/repos/cfpb/cfgov-refresh/node_modules/jquery/dist/jquery.js":34}],72:[function(require,module,exports){
```

Comment:
> is it worth considering using ES6 modules and then only using Browserify for older browsers? This way you eliminate some of the overhead associated with Browserify.

As far as I understand it, ES6 modules would be analogous to CommonJS, right? So switching to ES6 modules would affect how we authored the JS and just open up the possibility of using ES6 syntax that was polyfilled down, or transpiled using Babeljs. I think in terms of transpiling if a module had a pressing need for ES6 syntax it could do that and feed CommonJS into the rest of the system since Browserify does bundling of CommonJS modules. For alternatives, I've heard good things about webpack, which has page-specific bundling built-in, and SystemJS, which loads every module format. I went with CommonJS/Browserify because it seems like the bureau has standardized around that and OAH is using it so it'd make OAH integration less painful. It seems like we still have the issue of shimming non-modular 3rd-party libraries regardless if it was ES6, CommonJS, or AMD though.